awot/io/read_hiaper_radar.py

```diff
@@ -0,0 +1,214 @@
+"""
+    awot.io.read_haiper_radar
+    =================
+    
+    Scripts to read NCAR HIAPER Cloud Radar NetCDF data files.
+    
+    https://www.eol.ucar.edu/observing_facilities/hiaper-gulfstream-gv
+    
+    Testing was done from data file provided by Nguy.
+    """
+
+from netCDF4 import Dataset, num2date, date2num
+import datetime
+import numpy as np
+from . import common
+
+
+def read_hcr(fname, field_mapping=None, file_mapping=None):
+    '''
+        Read NCAR HIAPER Cloud Radar HCR NetCDF data file.
+        
+        ATTENTION!
+        There are extra variables that are ignored by the reader at this time.
+        
+        Parameters
+        ----------
+        fname : str
+        Filename
```

Comment:
Indent this second level 4 (i.e. Filename) spaces for pep8 compliance and readability.